.github/workflows/release-1x-integration.yaml

```diff
@@ -0,0 +1,61 @@
+name: Audio Integration Workflow 1.x
+
+on:
+  schedule:
+    - cron: '30 * * * *'
+
+env:
+ SELENIUM_GRID_PROVIDER: saucelabs
+ CLOUD_WATCH_METRIC: false
+ TEST_TYPE: Github-Action
+ SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
+ SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+
+jobs:
+  integ-audio:
+    name: Audio Integration Test 1.x
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Create a Job ID
+        id: create-job-id
+        uses: filipstefansson/uuid-action@ce29ebbb0981ac2448c2e406e848bfaa30ddf04c
+      - name: Set JOB_ID Env Variable
+        run: echo "JOB_ID=${{ steps.create-job-id.outputs.uuid }}" >> $GITHUB_ENV
+      - name: Echo Job ID
+        run: echo "${{ steps.create-job-id.outputs.uuid }}"
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Setup Sauce Connect
+        uses: saucelabs/sauce-connect-action@v1
+        with:
+          username: ${{ secrets.SAUCE_USERNAME }}
+          accessKey: ${{ secrets.SAUCE_ACCESS_KEY }}
+          noSSLBumpDomains: all
+          tunnelIdentifier: ${{ steps.create-job-id.outputs.uuid }}
+      - name: Checkout Package
```

Comment:
Could not understand these two checkouts `current` and `prev`? Could you please explain a little?

Reply:
No longer using this but the idea is to checkout previous repo in prev folder and current repo in current folder. The issue is i made some changes to run-test script but its not available in 2.8.0 so need to copy the file over.